atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueScopingCacheImpl.java

```diff
@@ -53,8 +54,11 @@ public final class LockWatchValueScopingCacheImpl implements LockWatchValueScopi
     private final ValueStore valueStore;
     private final SnapshotStore snapshotStore;
 
+    private final RateLimiter rateLimiter = RateLimiter.create(1 / 30.0);
```

Comment:
Can we use the `RateLimitedLogger` instead? It's a class designed exactly for this!

Reply:
also, what are your thoughts on maybe just 1 per second? Still a huge improvement over what we're seeing now!